reordering-streams/kstreams/src/main/java/io/confluent/developer/Event.java

```diff
@@ -0,0 +1,9 @@
+package io.confluent.developer;
+
+/**
+ * Bill Bejeck
+ * 6/7/24
+ */
```

Comment:
remove? did IDE add this?

Reply:
But I don't want to be forgotten! Ha!
Yep it's IDE generated, I'll have to disable for this repo